tsconfig.json

```diff
@@ -5,20 +5,7 @@
     "strict": true,
     "esModuleInterop": true,
     "outDir": "dist",
-    "declaration": true,
-    "skipLibCheck": true,
-    "forceConsistentCasingInFileNames": true,
+    "declaration": true
   },
-  "allowJs": false,
-  "include": [
-    "./scripts",
-    "./test",
```

Comment:
Was there a reason these config params were removed (ie `skipLibCheck`, `forceConsistentCasingInFileNames`, `"allowJs": false`, and `./test` in `include`)?

Or do they actually have no real use here, what with javascript type configuration being rubbish and all.

Reply:
I can't remember why these were added in here to be honest... Some of this looks like some old leftover stuff or possibly even junk copied over from fractal (haven't verified that).

Reply:
These configurations were largely taken from fractal with a couple of minor changes to fit our project structure